docs/reproin.rst

```diff
@@ -0,0 +1,139 @@
+================
+Reproin 
+================
+
+This tutorial is based on `Dianne Patterson's University of Arizona tutorials <https://neuroimaging-core-docs.readthedocs.io/en/latest/pages/heudiconv.html#lesson-3-reproin-py>`_
+
+`Reproin <https://github.com/ReproNim/reproin>`_ is a setup for
+automatic generation of sharable, version-controlled BIDS datasets from
+MR scanners.
+
+If can control how your image sequences are named at the scanner you can
+use the *reproin* naming convention.  
+
+Get Example Dataset
+-------------------
+
+This example uses a phantom dataset: `reproin_dicom.zip <https://osf.io/4jwk5/>`_ generated by the University of Arizona on their Siemens Skyra 3T with Syngo MR VE11c software on 2018_02_08.
+
+The ``REPROIN`` directory is a simple reproin-compliant DICOM (.dcm) dataset without sessions. 
+(Derived dwi images (ADC, FA etc.) that the scanner produced have been removed.::
+
+    [user@local ~/reproin_dicom/REPROIN]$ tree -I "*.dcm"
+
+        REPROIN
+        ├── data
+        └── dicom
+            └── 001
+                └── Patterson_Coben\ -\ 1
+                    ├── Localizers_4
+                    ├── anatT1w_acqMPRAGE_6
+                    ├── dwi_dirAP_9
+                    ├── fmap_acq4mm_7
+                    ├── fmap_acq4mm_8
+                    ├── fmap_dirPA_15
+                    └── func_taskrest_16
+
+Convert and organize
+--------------------
+
+From the ``REPROIN`` directory::
+
+    heudiconv -f reproin --bids  -o data --files dicom/001 --minmeta
+
+* ``-f reproin`` specifies the converter file to use
+* ``-o data/`` specifies the output directory *data*.  If the output directory does not exist, it will be created.
+* ``--files dicom/001`` identifies the path to the DICOM files.
+*  ``--minmeta`` ensures that only the minimum necessary amount of data gets added to the JSON file when created.  On the off chance that there is a LOT of meta-information in the DICOM header, the JSON file will not get swamped by it. fmriprep and mriqc are very sensitive to this information overload and will crash, so minmeta provides a layer of protection against such corruption.
```

Comment:
```suggestion
*  ``--minmeta`` ensures that only the minimum necessary amount of data gets added to the JSON file when created.  On the off chance that there is a LOT of meta-information in the DICOM header, the JSON file will not get swamped by it. Rumors are that fMRIPrep and MRIQC might be sensitive to excess of metadata and might crash crash, so minmeta provides a layer of protection against such corruption.
```
hm... I would like to investigate that and ideally remove from here.
Insofar I do not see any reports in https://github.com/search?q=repo%3Anipreps%2Ffmriprep+heudiconv&type=issues .